src/deferred.js

```diff
@@ -385,5 +394,12 @@ jQuery.extend( {
 	}
 } );
 
+function deferredTick() {
+	processQueueTimeout = undefined;
+	jQuery.each( processQueue.splice( 0 ), function( _, process ) {
```

Comment:
The `.splice(0)` clears the `processQueue`. That's why `splice` instead of `slice`. This does make the assumption that `process()` will never through anything though...